js/views/map.js

```diff
@@ -22,18 +22,41 @@ DeveloperPlayground.MapView = Backbone.View.extend({
         this.stopListening();
     },
 
-    
+
+    hasWebGL: function() {
+        //detect webgl is available on the browser
+        var canvas = document.createElement('canvas'),
+            gl,
+            glExperimental = false;
+
+
+        try { gl = canvas.getContext("webgl"); }
+        catch (x) { gl = null; }
+
+        if(gl) { return true; }
+        else { return false; }
+    },
+
     render: function() {
         this.markerclustergroup = new L.MarkerClusterGroup({showCoverageOnHover: false});
+
         this.map = L.map('map-view',{
             scrollWheelZoom: false
         }).setView([37.749, -122.443], 11);
-        // run setMapview inside of setview
-        L.tileLayer('https://{s}.tiles.mapbox.com/v3/randyme.li1lhlf0/{z}/{x}/{y}.png', {
-            maxZoom: 18,
-            attribution: 'Map data &copy; <a target="_blank" href="http://openstreetmap.org">OpenStreetMap</a> contributors, <a target="_blank" href="http://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA</a>, Imagery © <a target="_blank" href="http://mapbox.com">Mapbox</a>',
-        })
-        .addTo(this.map);
+
+        var layer;
+        if(this.hasWebGL()) {
+            var layer = Tangram.leafletLayer({
+                 scene: 'https://cdn.rawgit.com/tangrams/multiverse/gh-pages/styles/tangram-toner.yaml',
```

Comment:
I tried putting refill first, but it didn't look best. (attaching screenshot)
Waiting for Cartography team to get base map ready.  
* Playground will probably stick with raster tiles for now 
![screen shot 2015-10-08 at 4 30 19 pm](https://cloud.githubusercontent.com/assets/4583806/10378969/0ea48c8c-6dda-11e5-8975-b7d2b9614fa6.png)
 about base map